rollup.config.js

```diff
@@ -57,20 +57,33 @@ const defaultExternal = deps.concat(peerDeps);
 
 // We need to define 2 separate configs (`esm` and `cjs`) so that each can be
 // further customized.
-const config = {
-  input: 'src/index.js',
-  external: defaultExternal,
-  output: [
-    {
-      file: pkg.module,
-      format: 'esm',
-    },
-    {
-      file: pkg.main,
-      format: 'cjs',
-    },
-  ],
-  plugins,
-};
+const config = [
+  {
+    input: 'src/index.js',
+    external: defaultExternal,
+    output: [
+      {
+        file: pkg.module,
+        format: 'esm',
+      },
+      {
+        file: pkg.main,
+        format: 'cjs',
+      },
+    ],
+    plugins,
+  },
+  {
+    input: 'src/components/stamp/index.js',
+    external: defaultExternal,
+    output: [
+      {
+        file: 'dist/stamp.esm.js',
+        format: 'esm',
+      },
+    ],
+    plugins,
+  },
```

Comment:
How would we guide consumers to these entries?

Reply:
We could offer both the current [`big bundle`](https://www.youtube.com/watch?v=VDLDVPV8kDE), as well as the specific bundles. I posted above about writing a possible babel transform or something 🤔

Reply:
Yeah I get that. But when we publish small bundles to npm. How would consumers import them as the `module`, `main` etc all point to the bug bundle.

Reply:
They would need to either use the babel plugin or manually import the small bundles directly. Other libraries follow this pattern, I'll find an example soon

Reply:
Thanks.

Reply:
I tried to programmatically listing the exported components paths but I don't think it's possible. I guess we need to manually list the paths here?
We should probably find a better solution, for example we generate the main export file from a list, which we can use to define the exports bundle here as well.